understanding/predictable.html

```diff
@@ -54,64 +52,45 @@
 
-         <!-- 日本語訳されるまでの一時的な注記 -->
-         <div class="untranslated-note"><p>このWCAG 2.2 解説書の日本語訳は作業中となっています。WCAG 2.1 解説書の日本語訳をご利用いただけます。<br>[ <a href="https://waic.jp/translations/WCAG21/Understanding/">WCAG 2.1 解説書</a> ]</p></div>
-            <h1><span class="standalone-resource__type-of-guidance">Understanding 
-                  				Guideline 3.2:</span>Predictable
+            <h1><span class="standalone-resource__type-of-guidance">解説書 
+                  				ガイドライン 3.2:</span>予測可能
             </h1>
             <aside class="box">
-               <header class="box-h  box-h-icon">Guideline </header>
+               <header class="box-h  box-h-icon">ガイドライン</header>
                <div class="box-i">
-                  <p>Make Web pages appear and operate in predictable ways.</p>
+                  <p>ウェブページの表示や挙動を予測可能にすること。</p>
                </div>
             </aside>
             <div class="excol-all"></div>
             <section id="intent">
-               <h2>Intent</h2>
-               <p>The intent of this Guideline is to help users with disabilities by presenting content
-                  in a predictable order from Web page to Web page and by making the behavior of functional
-                  and interactive components predictable. It is difficult for some users to form an
-                  overview of the Web page: screen readers present content as a one-dimensional stream
-                  of synthetic speech that makes it difficult to understand spatial relationships. Users
-                  with cognitive limitations may become confused if components appear in different places
-                  on different pages.
+               <h2>意図</h2>
+               <p>このガイドラインの意図は、ウェブページからウェブページへの予測可能なコンテンツの順序を提示することによって、及び機能的でインタラクティブなコンポーネントの動作を予測可能にすることによって、障害のある利用者を支援することである。利用者がウェブページ全体の概観を把握するのが困難なことがある。例えば、スクリーンリーダーは、合成音声の 1 次元のストリームとしてコンテンツを提示するため、空間的な関係性を理解しづらくする。認知能力が低下した利用者は、コンポーネントがページによって異なる場所に表示される場合に混乱することがある。
                   
                </p>
-               <p>For example, people who use screen magnifiers see only part of the screen at any point
-                  in time; a consistent layout makes it easier for them to find navigation bars and
-                  other components. Placing repeated components in the same relative order within a
-                  set of Web pages allows users with reading disabilities to focus on an area of the
-                  screen rather than spending additional time decoding the text of each link. Users
-                  with limited use of their hands can more easily determine how to complete their tasks
-                  using the fewest keystrokes.
+               <p>例えば、画面拡大ソフトを使用している利用者は、ある時点で画面の一部分だけしか見ていない。一貫性のあるレイアウトによって、ナビゲーションバーやその他のコンポーネントが見つけやすくなる。ウェブページ一式の中で、繰り返し用いられるコンポーネントを同じ相対的順序で配置することにより、読字障害のある利用者が、各リンクのテキストをデコードするのに追加の時間を費やすことなく、画面のある領域に集中することを可能にする。手を十分に使えない利用者は、最少のキーストロークでタスクを完了できる方法をより容易に見つけ出すことができる。
```

Comment:
テキストを「デコード」する、というのはユーザーをマシン扱いしているような。ここでは読字障害のある利用者が自らテキストを読み取って理解すると言うことだと思うので、「読み解く」というくらいでしょうか。

```suggestion
               <p>例えば、画面拡大ソフトを使用している利用者は、ある時点で画面の一部分だけしか見ていない。一貫性のあるレイアウトによって、ナビゲーションバーやその他のコンポーネントが見つけやすくなる。ウェブページ一式の中で、繰り返し用いられるコンポーネントを同じ相対的順序で配置することにより、読字障害のある利用者が、各リンクのテキストを読み解くことに追加の時間を費やすことなく、画面のある領域に集中することを可能にする。手を十分に使えない利用者は、最少のキーストロークでタスクを完了できる方法をより容易に見つけ出すことができる。
```